### ISSUE TITLE
Can't re render inline editor content when tag is changes

### ISSUE THREAD

**Issue by azakusilo** [tinymce/tinymce-vue#96]
**What is the current behavior?**
I wrote button, which is changes the tag which is wrapping inline editor (h1, h2, etc,).
And when I change tag. Tag is changing, but editor has no content in it.
This is component HTML init:
`<tinymce-editor inline :id="getAtomTagId()" ref="editorLink" :class="getTinyClasses()" :initial-value="getInitialValue" :style="getStyle" :tag-name="tagName" :init="tinyMCEOptions" @onInit="onInitTinyMCE" @onBlur="onBlurTinyMCE"></tinymce-editor>`

data(){
  return {
    tagName: 'h1'
  }
}

Tag is dynamically string in component. I change h1 to h2 and etc.
This is the button which is have change function:
```
tinyMCEOptions: {
                menubar: false,
                language: 'ru',
                plugins: [
                    'link',
                    'lists',
                    'autolink'
                ],
                toolbar: [
                    'bold italic underline fontselect fontsizeselect link unlink forecolor backcolor selectTags'
                ],
                fontsize_formats: '10px 12px 14px 16px 18px 20px 22px 24px 26px 28px 30px 32px 34px 36px 38px 40px 42px 44px 46px 48px 50px 52px 54px 56px 58px 60px 62px 64px 66px 68px 70px',
                setup: (editor) => {
                    let data = this.getData(); //Object with tags
                    editor.ui.registry.addMenuButton('selectTags', {
                        text: data.tag.toString().toLocaleUpperCase(),
                        fetch: (callback) => {
                            let data = this.getData();
                            let tags = ['h1', 'h2', 'h3'];  // ['h1', 'h2', 'h3']
                            let items = [];

                            //Пропишем смену тегов
                            tags.forEach((tag) => {
                                items.push({
                                    type: 'menuitem',
                                    text: tag.toString().toLocaleUpperCase(),
                                    onAction: async () => {
                                       this.tagName = tag;
                                    }
                                })
                            });
                            callback(items);
                        }
                    });
                }
            }
```

**What is the expected behavior?**
content must to been showed with new tag.

**Which versions of TinyMCE, and which browser / OS are affected by this issue? Did this work in previous versions of TinyMCE or tinymce-vue?**
5.0.14, Window 8

**Comment by SimonFc** [comment]
It sounds like you have a question about implementation? I can help you with that but could you set it up on [CodeSandbox](https://codesandbox.io/s/9y95z5m40y?fontsize=14)? It's difficult to follow without all the code

**Comment by azakusilo** [comment]
Here you are:
[https://codesandbox.io/s/tinymcevue-zmp0j?fontsize=14](url)
Try to change tag in new button selector.

**Comment by SimonFc** [comment]
There seems to be an issue with tinymce-vue not updating to reflect changes to certain props. It might or might not be a bug and it's going to take some time to investigate.
As for your specific case though, do you need to initialize directly on a heading?

**Comment by azakusilo** [comment]
Yes, it is. I need to change tag directly. It changes, but content is lost.

**Comment by azakusilo** [comment]
Hello. Is have any news for this issue?

**Comment by SimonFc** [comment]
Unfortunately not yet altough we're tracking it in our internal issue tracker

**Comment by azakusilo** [comment]
Hello. Is have any new for this issue for now?

**Comment by seedprod** [comment]
I solved this by adding a key to the component and watch for when the tag changes, then I increment the component key and vue rerenders with the correct tag.

**Comment by azakusilo** [comment]
> I solved this by adding a key to the component and watch for when the tag changes, then I increment the component key and vue rerenders with the correct tag.

Exelent!

**Comment by jscasca** [comment]
This issue has been fixed in v4.0.3 by adding a listener to the TagName attribute. This case can be found in the package's vite demo.
